src/main/kotlin/com/github/suusan2go/kotlinfillclass/inspections/FillClassInspection.kt

```diff
@@ -57,6 +60,7 @@ class FillClassInspection(
         val panel = MultipleCheckboxOptionsPanel(this)
         panel.addCheckbox("Fill arguments without default values", "withoutDefaultValues")
         panel.addCheckbox("Do not fill default arguments", "withoutDefaultArguments")
+        panel.addCheckbox("Append trailing comma", "withTrailingComma")
```

Comment:
Note: the IDE flags this API

>'addCheckbox(java.lang.@org.jetbrains.annotations.NotNull @com.intellij.openapi.util.NlsContexts.Checkbox String, java.lang.@org.jetbrains.annotations.NotNull @org.jetbrains.annotations.NonNls String)' is available only since 213.4928.7 but the module is targeted for 191.8026.42 - 221.SNAPSHOT. It may lead to compatibility problems with IDEs prior to 213.4928.7. Note that this method might have had a different full signature in the previous IDEs.